extensions/objects/templates/coord-two-dim-editor.component.ts

```diff
@@ -19,7 +19,14 @@
 import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
 import { downgradeComponent } from '@angular/upgrade/static';
 import { UrlInterpolationService } from 'domain/utilities/url-interpolation.service';
-import { icon, latLng, LeafletEvent, LeafletMouseEvent, marker, tileLayer } from 'leaflet';
+import { icon, LatLng, latLng, LeafletEvent, LeafletMouseEvent, Marker, marker, TileLayer, tileLayer } from 'leaflet';
+
+interface Options {
+  layers: TileLayer[];
+  zoom: number;
+  center: LatLng;
+}
```

Comment:
Should this name be more specific?

Reply:
I missed this, @mridul-netizen you can actually import `MapOptions` from `leaflet` and use it as a type for `options` instead.

Reply:
Done thanks, @vojtechjelinek PTAL